src/commands/assistants/Issue.js

```diff
@@ -0,0 +1,37 @@
+const {Command} = require('discord.js-commando')
+
+module.exports = class LmgtfyCommand extends Command {
```

Comment:
Perhaps change the class to IssueCommand.

Reply:
Nice catch!